CONTRIBUTING.md

```diff
@@ -0,0 +1,109 @@
+# Contributing to PyPowerwall
```

Comment:
I'm not sure the right way to do this, but we should also note that PyPowerwall is almost never used without Powerwall-Dashboard, the two are pretty inextricably linked sister projects. So that should be mentioned some way, some how.

Another thought is: Often bugs in one migrate to the other and vice versa, how do we want to handle that?

Finally: What is the difference between an issue and a discussion? When should one we used or not? When do we close an issue or a discussion? Obviously there will never be hard-fast rules here, but there could be guidelines.

Reply:
I like what you wrote!